src/app/classes/classes.component.html

```diff
@@ -1,15 +1,43 @@
-<h2>Class List</h2>
-<ul class="classes" style="list-style-type: none">
-  <li
-    *ngFor="let class of classes"
-    [class.selected]="class === selectedClass"
-    (click)="onSelect(class)"
-  >
-    {{ class.subject }} {{ class.course }}
-  </li>
-</ul>
-
-<div *ngIf="selectedClass">
-  <h2>Subscribed to {{ selectedClass.subject }} {{ selectedClass.course }}</h2>
-  <div>{{ selectedClass.id }}</div>
+<div class="container">
+  <div class="form-group">
+    <label for="search-text">Search Your Class</label>
+    <input
+      type="email"
```

Comment:
Why type="email", should it be type="text"

Reply:
type fixed